app/controllers/home_controller.rb

```diff
@@ -1,2 +1,2 @@
 class HomeController < ApplicationController
```

Comment:
```suggestion
class HomeController < ApplicationController
       def index; end
```
You should have the index method